packages/accordion-react/integration/accordion.spec.js

```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress" />
+
+context("Accordion", () => {
+    beforeEach(() => {
+        cy.visit("/komponenter/accordion");
+    });
+
+    it("Accordion should work", () => {
+        cy.get("[data-testid=jkl-accordion]").toMatchImageSnapshot();
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.get("[data-testid=jkl-accordion]").wait(100).toMatchImageSnapshot();
```

Comment:
Denne er bare er for å illustrere en bug i accordion.

Normalt sett trenger man ikke explisitte waits i Cypress.